manim/mobject/geometry.py

```diff
@@ -1554,3 +1556,228 @@ def __init__(self, main_shape, *mobjects, **kwargs):
+
+    Examples
+    --------
+    The first example shows some right angles with a dot in the middle while the second example shows
+    all 8 possible angles defined by two lines.
+
+    ..manim:: RightArcAngleExample
+        :save_last_frame:
+
+        class RightArcAngleExample(Scene):
+            def construct(self):
+                line1 = Line( LEFT, RIGHT )
+                line2 = Line( DOWN, UP )
+                rightarcangles = [
+                    ArcAngle(line1, line2, dot=True),
+                    ArcAngle(line1, line2, radius=0.4, quadrant=(1,-1), dot=True, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8, dot=True, dot_color=YELLOW, dot_radius=0.04, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED, dot=True, dot_color=GREEN, dot_radius=0.08),
+                ]
+                line_list = VGroup( *[VGroup() for k in range(4)] )
+                for k in range(4):
+                    linea = line1.copy()
+                    lineb = line2.copy()
+                    line_list[k].add( linea )
+                    line_list[k].add( lineb )
+                    line_list[k].add( rightarcangles[k] )
+                line_list.arrange_in_grid(buff=1.5)
+                self.add(
+                    line_list
+                )
+
+    ..manim:: ArcAngleExample
+        :save_last_frame:
+
+        class ArcAngleExample(Scene):
+            def construct(self):
+                line1 = Line( LEFT + (1/3) * UP, RIGHT + (1/3) * DOWN )
+                line2 = Line( DOWN + (1/3) * RIGHT, UP + (1/3) * LEFT )
+                arcangles = [
+                    ArcAngle(line1, line2),
+                    ArcAngle(line1, line2, radius=0.4, quadrant=(1,-1), other_angle=True),
+                    ArcAngle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED),
+                    ArcAngle(line1, line2, other_angle=True),
+                    ArcAngle(line1, line2, radius=0.4, quadrant=(1,-1)),
+                    ArcAngle(line1, line2, radius=0.5, quadrant=(-1,1), stroke_width=8),
+                    ArcAngle(line1, line2, radius=0.7, quadrant=(-1,-1), color=RED, other_angle=True),
+                ]
+                line_list = VGroup( *[VGroup() for k in range(8)] )
+                for k in range(8):
+                    linea = line1.copy()
+                    lineb = line2.copy()
+                    line_list[k].add( linea )
+                    line_list[k].add( lineb )
+                    line_list[k].add( arcangles[k] )
+                line_list.arrange_in_grid(n_rows=2, n_cols=4, buff=1.5)
+                self.add(
+                    line_list
+                )
+
+    """
+
+    def __init__(
+        self,
+        line1,
+        line2,
+        radius=0.3,
```

Comment:
```suggestion
        radius=None,
```
I'd prefer this default to be `None` and calculated dynamically based on the size of the shorter of the two segments from the intersection. See this example:
```py
class Test(Scene):
    def construct(self):
        l1 = Line(ORIGIN, RIGHT)
        l2 = Line(ORIGIN, UP*.2)
        a = ArcAngle(l1, l2)
        self.add(l1,l2,a)
```
![image](https://user-images.githubusercontent.com/29168469/110840115-32bed700-8259-11eb-9c5c-d7e040b19911.png)

Reply:
Would you say something like the smaller of 75% of the shorter side or 0.3? Or would two large lines produce a large angle by default?

Reply:
I think 75%, 50%, or 66.66666% are sensible for the shorter side. I also think two long line subsections should produce a large angle (radius from the "intersection", not theta) by default. Perhaps there can be hard cap (i.e. flooring) for the radius size as well. It's just awkward that the default radius will sometimes not hit the line(s).

Reply:
I did implement that with a hard cap of 0.4 as the biggest radius. Other than that the radius will be 2/3 of the shorter line. Thanks for the idea!